public/components/common/search/search.tsx

```diff
@@ -259,7 +259,6 @@ export const Search = (props: any) => {
       setSelectedIndex(reduxIndex);
       // sets the editor text and populates sidebar field for a particular index upon initialization
       const indexQuery = `source = ${reduxIndex[0].label}`;
-      handleQueryChange(indexQuery);
```

Comment:
If I'm remembering correctly, this method was used to set the query inside of the "query area" from a redirection related to query assistant. Did we know what this was used for beforehand and if it remains unaffected with this change?
One of these redirections I believe were from the welcome page with sample indices and sample natural language queries, could start with testing that interaction.

Reply:
@paulstn Do you know was it some custom welcome page or released in a particular OS version? We are not able to find this on our side..

Reply:
Tested the 2.13 new home page setting which just provides links to documentation, no workflow identified that landed on the explorer page with an example query.

Reply:
Regardless of being able to find the welcome page, the purpose of this line was to populate the query area with some placeholder text. If we're fine removing that placeholder then there isn't an issue.